### DISCUSSION TITLE
Monitoring resold power ?

### DISCUSSION THREAD

**Discussion by MoltesDu12** [addiejanssen/domoticz-solaredge-modbustcp-plugin#24] [Ideas]
Hello,
I use this plugin for a few week. It's a very good job !
But I noticed that this plugin doesn't show how many power is resold to my provider.
Don't you planed to integrate resold power ?

**Comment by addiejanssen** [comment]
Hi,

The main purpose of the plugin is to see what the solar pannels that are attached to the inverter are producing.
Over time, I learned that some of the Solaredge inverters have an ability to attach a power meter and batteries.

In discussion #13 read meter data,  Attila Gröpler added a zip file with a modified version of the plugin that enables the usage of a power meter in/attached to the inverter.
I have not included the modifications in the plugin, since I have no way to test it myself.

If that is what you are looking for: give it a spin and let us know if it works.

And if there is somebody out there that is willing to open up their inverter to me (I can share my ip address to restrict access), then I am more than happy to try to integrate this in the plugin.

**Reply by MoltesDu12** [reply]
I tested the script provided by Attila and it works ! 
Thanks !
